code/modules/psionics/faculties/energistics.dm

```diff
@@ -7,24 +7,27 @@
 /datum/psionic_power/energistics
 	faculty = PSI_ENERGISTICS
 
-/datum/psionic_power/energistics/disrupt
-	name =            "Disrupt"
-	cost =            10
+/datum/psionic_power/energistics/sequencer
```

Comment:
I do not believe that infinite emag is a good idea, due to the difficulty involved in undoing the damage caused by it / balance concerns

Reply:
Alright, fair. I'll make up something to replace it.

Reply:
This is addressed.